src/main.py

```diff
@@ -73,13 +73,35 @@ def main() -> NoReturn:
     )
     config_file = path.join(prog_folder, "config.ini")
     log_file = path.join(prog_folder, "scanner.log")
-    logging.basicConfig(
-        format="%(asctime)s %(levelname)-8s %(message)s",
-        level=logging.INFO,
+
+    # Remove all handlers
+    for handler in logging.root.handlers:
+        logging.root.removeHandler(handler)
+
+    logging.root.setLevel(logging.INFO)
+    # Define stream formatter and handler
+    stream_formatter = colorlog.ColoredFormatter(
+        fmt="[%(cyan)s%(asctime)s%(reset)s][%(log_color)s%(levelname)s%(reset)s] - %(message)s",
```

Comment:
```suggestion
        fmt="%(cyan)s%(asctime)s%(reset)s %(log_color)s%(levelname)-8s%(reset)s %(message)s",
```
Just a small formatting change. I prefer that the log messages are aligned so the log is more readable.
That's not possible for the logfile due to the function and module names of course.